.gitignore

```diff
@@ -119,3 +123,5 @@ crates/glue/tests/fixtures/*/build.rs
 crates/glue/tests/fixtures/*/host.c
 crates/glue/tests/fixtures/*/src/main.rs
 crates/glue/tests/fixtures/*/test_glue/
+
+**/*platform/glue
```

Comment:
We include a new crate/script to copy the zig glue files into each of the test platforms before a test run. This is a workaround for creating a zig source package out of the builtins, and upgrading all of the test platform hosts to use the full zig build system. 

Instead this enables us to keep the simpler cli `zig build-obj` etc.